sklearn/calibration.py

```diff
@@ -33,25 +33,31 @@
 from .utils.validation import _deprecate_positional_args
 
 
-def _fit_calibrated_classifer(estimator, X, y, train, test, supports_sw,
-                              method, classes, sample_weight=None):
+def _fit_calibrated_classifer(estimator, X, y, train, test,
+                              method, classes, sample_weight=None,
+                              base_estimator_uses_sw=False, **fit_params):
```

Comment:
I would avoid to pass `base_estimator_uses_sw`. I think that we can make the check within the function itself again.

Reply:
:+1: